src/PQDashboard/Scripts/Default.js

```diff
@@ -3929,35 +3929,22 @@ function loadLeafletMap(theDiv) {
 }
 
 function addEsriLayers() {
-    if (arcGis.BaseUri == "" || arcGis.ServiceLayers == "") return;
+    if (arcGis.BaseUri == "") return;
 
     const baseURI = new URL("/arcgis/rest/services/" + arcGis.Folder + "/" + arcGis.ServiceName + "/MapServer", arcGis.BaseUri).toString();
-    const layers = JSON.parse(arcGis.ServiceLayers);
-    let gisLayers = {};
-
-    L.Icon.Default.imagePath = "Scripts/Leaflet/images";
-    layers.forEach(function (layer) {
-        let layerProperties = {};
-        layerProperties['url'] = baseURI + "/" + layer.ID;
-        if (layer.Image != "")
-            layerProperties['pointToLayer'] =
-                function (geojson, latlng) {
-                    return L.marker(latlng, {
-                        icon: L.icon({
-                            iconUrl: layer.Image
-                        })
-                    });
-                };
-        if (layer.Color != "")
-            layerProperties['style'] =
-                function () {
-                    return {
-                        color: layer.Color
-                    }
-                };
-        gisLayers[layer.Name] = L.esri.featureLayer(layerProperties);
-    });
-    L.control.layers(null, gisLayers).addTo(leafletMap[currentTab]);
+
+    $.getJSON(baseURI + "/layers?f=pjson&callback=?",
```

Comment:
seems a little strange that you would need  include `?f=jpjson&callback=?` can you double check these are required?
I'd think `/layers?f=pjson` would be sufficent if anything

Reply:
It is required. Otherwise, getJSON assumes a regular json and not a jsonp, which gets blocked by CORS

Reply:
You are correct:
https://api.jquery.com/jquery.getjson/
Still seems strange to me but ok will merge.